okta/api_response.py

```diff
@@ -136,11 +135,7 @@ async def next(self):
         if self._type is not None:
             result = []
             for item in next_page:
-                result.append(
-                    self._type(
-                        APIClient.form_response_body(item)
-                    )
-                )
+                result.append(self._type(item))
```

Comment:
Removing the call to re-format the response body in the `next()` method which returns the paginated response

Also done like this in resource clients:
https://github.com/okta/okta-sdk-python/blob/master/okta/resource_clients/user_client.py#L112-L114